src/kundera-rdbms/src/main/java/com/impetus/client/rdbms/query/RDBMSQuery.java

```diff
@@ -75,6 +75,7 @@ public RDBMSQuery(KunderaQuery kunderaQuery, PersistenceDelegator persistenceDel
             final KunderaMetadata kunderaMetadata)
     {
         super(kunderaQuery, persistenceDelegator, kunderaMetadata);
+        setMaxResults(0);
```

Comment:
By default, maxResults is 100 in Kundera. In MongoDB, you put `Interger.MAX_VALUE` if user explicitly set it to 0. But in RDBMS, you are changing default behavior and setting it to 0.  It should be consistent among different modules of Kundera if  user has not set maxResults explicitly.